shark/billing/admin.py

```diff
@@ -81,8 +57,20 @@ class InvoiceAdmin(admin.ModelAdmin):
     list_filter = ("created_at", "paid_at", "type")
     date_hierarchy = "created_at"
     actions = ("total_value_action", "export_for_accounting")
-    save_on_top = True
-    autocomplete_fields = ("customer",)
+    raw_id_fields = ("customer",)
```

Comment:
That's a situation where [django-autocomplete-light](https://pypi.org/project/django-autocomplete-light/) is really nice to have.